### PR TITLE
Bv menus

### PR DESCRIPTION
**Pull request by Willyfrog**
**Summary**

Most menus working, there are a few that still need some extra work

### CODE REVIEW

**Comment by nevyangelova** [comment]
Can we rebase with the feature branch?

**Comment by Willyfrog** [comment]
merged to be up-to-date with the feature branch, so it is ready to review :)

File: src/renderer/index.jsx [hunk 1]
```diff
@@ -18,202 +17,144 @@ if (process.env.NODE_ENV === 'production') {
...
+    reloadConfig(configRequest);
+  } catch (err) {
+    if (exitOnError) {
+      ipcRenderer.send(QUIT, `unable to load configuration: ${err}`, err.stack);
+    }
+  }
+};
 
-remote.getCurrentWindow().removeAllListeners('focus');
+const start = async () => {
```
Comment (devinbinnie) [inline src/renderer/index.jsx:55]: Nit: Move this to the bottom of the file where the rendered component would likely be.

File: src/renderer/settings.jsx [hunk 2]
```diff
@@ -2,61 +2,40 @@
...
-
-function setDarkMode() {
-  if (process.platform !== 'darwin') {
-    const darkMode = Boolean(config.darkMode);
-    config.set('darkMode', !darkMode);
-    return !darkMode;
-  }
-  return null;
-}
+const start = async () => {
```
Comment (devinbinnie) [inline src/renderer/settings.jsx:22]: Nit: same here

**Review by devinbinnie — APPROVED** [review]

**Review by nevyangelova — APPROVED** [review]
LGTM :)

**Pull request merged**